app/app.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/rancher/rancher-compose-executor/project/options"
 	"github.com/rancher/rancher-compose-executor/rancher"
 	"github.com/urfave/cli"
+
```

Comment:
Nitpick: These imports are organized poorly. golang system packages should go at the top, separated by a newline from user imports, which should be grouped together with no newlines.